pybullet_toolkit/robots/ur5_rg2.py

```diff
@@ -77,8 +92,8 @@ def place(self, pos, rot, offset, dynamic=True):
     pre_rot = pb.getQuaternionFromEuler([0, np.pi, 0])
 
     # Move to pre-grasp pose and then grasp pose
-    self.moveTo(pre_pos, pre_rot, dynamic)
-    self.moveTo(pos, rot, dynamic)
+    self.moveTo(pre_pos, pre_rot, True)
```

Comment:
Might as well keep these as the dynamic flag rather than set them to True. Doesn't cost us anything and it is more clear/easier to add static in the future.